Rakefile

```diff
@@ -5,3 +5,17 @@ require File.expand_path("../config/application", __FILE__)
 
 Rails.application.load_tasks if Rake::Task.tasks.empty?
 KnapsackPro.load_tasks if defined?(KnapsackPro)
+
+if Rails.env.development?
+  require "github_changelog_generator/task"
+
+  GitHubChangelogGenerator::RakeTask.new :changelog do |config|
+    config.user = "consul"
+    config.project = "consul"
+    config.since_tag = "1.0.0"
+    config.future_release = "1.1.0"
+    config.base = "#{Rails.root}/CHANGELOG.md"
+    config.token = Rails.application.secrets.github_changelog_token
+    config.max_issues = 1
```

Comment:
In the past, we had `max_issues = 1` in the Rakefile and `max_issues = 0` in the `.github_changelog_generator` file. I don't know which way has precedence :thinking:. Is it this one?